.github/workflows/pixi.yml

```diff
@@ -0,0 +1,21 @@
+name: Pixi
+
+on:
+  pull_request:
+  push:
+    branches: [main]
```

Comment:
Is it possible to use a files section like in #54 so that it's only checked when the pixi files are updated?

Reply:
derp, like #52

Reply:
I think this should be checked when any file is edited because it is going to build and run the tests in this environment.

Reply:
Oh, after looking at your link I understand, updated.

Reply:
yup, perfect. Thanks!